### PR TITLE
feature/update-label

### PR DESCRIPTION
**Pull request by zuzannaszymanda**

### CODE REVIEW

**Comment by szafran89** [comment]
Please check unit test issue

File: src/atoms/label/Label.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (szafran89) [inline src/atoms/label/Label.css]: remove this empty line. I will add some lint for this file

File: src/atoms/label/Label.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 export default {
   props: {
     /**
-     * Label tag
+     * To use another tag instead of 'label'
      */
     tag: {
```
Comment (szafran89) [inline src/atoms/label/Label.js]: IMO custom tag is unnecessary, WDYT is there any use case for custom tag?

File: src/atoms/label/Label.vue [hunk 3]
```diff
@@ -10,4 +10,4 @@ export default {
 }
 </script>
 
-<style lang="scss" src="./Label.scss" />
+<style lang="css" src="./Label.css" />
```
Comment (szafran89) [inline src/atoms/label/Label.vue]: please remove lang attr

File: src/atoms/label/Label.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+.a-label {
+  @apply block;
+  @apply mb-1;
+  @apply text-gray-600 font-normal text-base font-sans leading-relaxed;
+  @apply transition-none;
```
Comment (szafran89) [inline src/atoms/label/Label.css]: Hmm.. do we need this line?

File: src/atoms/label/Label.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
+
+.a-label {
+  @apply block;
+  @apply mb-1;
+  @apply text-gray-600 font-normal text-base font-sans leading-relaxed;
```
Comment (szafran89) [inline src/atoms/label/Label.css:5]: I'm not sure all properties are needed. Ca we remove text-base, font-sans and font-normal?

File: src/atoms/label/Label.js (hunk 2 above)
Comment (zuzannaszymanda) [reply src/atoms/label/Label.js]: not really, I'll change component to label

**Review by szafran89 — APPROVED** [review]

**Pull request merged**